org.seventhstring.Transcribe.yml

```diff
@@ -0,0 +1,40 @@
+app-id: org.seventhstring.Transcribe
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: transcribe
+finish-args:
+#TODO: figure out if its possible to only grant permissions 
+# on ~/.Transcribe!7 and xdg-music:ro, for now we use home
+# as a functional compromise
+- --share=ipc
+- --socket=x11
+- --socket=wayland
+- --socket=pulseaudio
+- --filesystem=home
+modules:
+  - name: transcribe
+    buildsystem: simple
+    build-commands: 
+    - mkdir -p /app/bin
+    - cp transcribe libgstvideosection.so xschelp.htb /app/bin
+    - mkdir -p /app/share/icons/hicolor/16x16/apps && mv gtkicons/xsc16x16.png /app/share/icons/hicolor/16x16/apps/org.seventhstring.Transcribe.png
+    - mkdir -p /app/share/icons/hicolor/32x32/apps && mv gtkicons/xsc32x32.png /app/share/icons/hicolor/32x32/apps/org.seventhstring.Transcribe.png
+    - mkdir -p /app/share/icons/hicolor/48x48/apps && mv gtkicons/xsc48x48.png /app/share/icons/hicolor/48x48/apps/org.seventhstring.Transcribe.png
+    - mkdir -p /app/share/icons/hicolor/128x128/apps && mv gtkicons/xsc128x128.png /app/share/icons/hicolor/128x128/apps/org.seventhstring.Transcribe.png
+    - |
+      mkdir -p /app/share/applications && echo "
+      [Desktop Entry]
+      Encoding=UTF-8
+      Version=1.0
+      Type=Application
+      Exec=/app/bin/transcribe
+      Categories=Audio;
+      Terminal=false
+      Icon=org.seventhstring.Transcribe
+      Name=Transcribe!
+      " > /app/share/applications/org.seventhstring.Transcribe.desktop
+    sources:
```

Comment:
There is no appstream file. One is required.

Reply:
will do!

Reply:
actually this is why I was asking about using the install script. Because otherwise this will be out of sync from upstream.

Reply:
Sorry, i did not understand that, could you explain again please?

Reply:
wrong thread

Reply:
added a WIP appstream file ( missing description still )